base/file.jl

```diff
@@ -673,5 +680,4 @@ is `-1` the corresponding ID will not change. Only integer `owner`s and `group`s
 function chown(path::AbstractString, owner::Integer, group::Integer=-1)
     err = ccall(:jl_fs_chown, Int32, (Cstring, Cint, Cint), path, owner, group)
     uv_error("chown",err)
-    nothing
```

Comment:
Maybe `chmod` and `chown` should return the path too?

Reply:
done